hydro/exchange/merge.py

```diff
@@ -2,18 +2,25 @@
 from typing import Hashable, Tuple
 
 from .containers import Exchange
+from .exceptions import ExchangeInconsistentMergeType
 
 
 def ordered_merge(*tuples: Tuple[Hashable, ...]):
     return tuple(dict.fromkeys(chain(*tuples)).keys())
 
 
 def merge_ex(*excahnges: Exchange) -> Exchange:
+
+    if len({ex.file_type for ex in excahnges}) != 1:
```

Comment:
Persistent typo on exchanges?

Reply:
Heh... probably, I just autocomplete in my IDE